lib/board.rb

```diff
@@ -0,0 +1,55 @@
+class Board
+
+	attr_accessor :cells
+
+	def initialize
+		reset!
```

Comment:
Hm. Did you set the preferences to use 2 spaces instead of tabs?

On Tue, May 3, 2016 at 8:44 PM Amy <notifications@github.com> wrote:

> In lib/board.rb
> <https://github.com/learn-co-students/ttt-with-ai-project-v-000/pull/134#discussion_r61979371>
> :
>
> > @@ -0,0 +1,55 @@
> > +class Board
> > +
> > +	attr_accessor :cells
> > +
> > +	def initialize
> > +		reset!
>
> I'm using Sublime 3 on a Mac.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/learn-co-students/ttt-with-ai-project-v-000/pull/134/files/c1329a9310498664ad6eac2bce2945a1a3249547#r61979371>
>